test/externalTests.sh

```diff
@@ -35,6 +35,10 @@ REPO_ROOT=$(realpath "$(dirname "$0")/..")
 
 verify_input "$@"
 
```

Comment:
You can simply move your changes to the top of `test/externalTests/common.sh` for this to work.

Reply:
I don't think it is possible to do it inside `verify_input`, but because the scripts use `source` on `common.sh` it works the way you implemented it if you move your changes to, say, line 26 of `common.sh`.

Reply:
Ah, right. I didn't think of that!